flutter_secure_storage/example/integration_test/app_test.dart

```diff
@@ -75,6 +78,130 @@ void main() {
         ..verifyRowDoesNotExist(0)
         ..verifyRowDoesNotExist(1);
     });
+
+    testWidgets('Enclave requested on iOS Simulator falls back gracefully',
+        (WidgetTester tester) async {
+      if (!(Platform.isIOS &&
+          Platform.environment.containsKey('SIMULATOR_DEVICE_NAME'))) {
+        return; // Skip when not running on iOS Simulator
+      }
```

Comment:
let me know what you think about this
```suggestion
    testWidgets('Enclave requested on iOS Simulator falls back gracefully',
        skip: !(Platform.isIOS && Platform.environment.containsKey('SIMULATOR_DEVICE_NAME')),
        (WidgetTester tester) async {
```